src/plugins/timeConductor/TimePopupFixed.vue

```diff
@@ -182,13 +182,22 @@ export default {
     this.handleNewBounds = _.throttle(this.handleNewBounds, 300);
   },
   mounted() {
+    document.addEventListener('keydown', this.handleKeyDown);
     this.setTimeSystem(JSON.parse(JSON.stringify(this.openmct.time.getTimeSystem())));
     this.setViewFromBounds(this.bounds);
   },
   beforeUnmount() {
     this.clearAllValidation();
+    document.removeEventListener('keydown', this.handleKeyDown);
   },
   methods: {
+    handleKeyDown({ key }) {
+      if (key === 'Enter' && !this.hasInputValidityError) {
```

Comment:
There seems to be no way to target the [X] button in the Time popup to hit Enter and cancel the action. 
The check button action works fine on Enter.

Reply:
Yeah it's not setup in that way, it will cancel if you press the 'Escape' key and submit if you hit the 'Enter' key